operations/firmware/Update_Firmware_with_FAS.md

```diff
@@ -120,51 +121,52 @@ After identifying which hardware is in the system, start with the top most item
 
 <a name="firmware-actions"></a>
 
-### Firmware Actions
+## Firmware Actions
 
 An action is collection of operations, which are individual firmware update tasks. Only one FAS action can be run at a time. Any other attempted action will be queued. Additionally, only one operation can be run on a component name (xname) at a time. For example, if there are 1000 component names (xnames) with 5 targets each to be updated, all 1000 component names (xnames) can be updating a target, but only 1 target on each component name (xname) will be updated at a time.
 
 The life cycle of any action can be divided into the static and dynamic portions of the life cycle.
 
 The static portion of the life cycle is where the action is created and configured. It begins with a request to create an action through either of the following requests:
 
--   Direct: Request to /actions API.
--   Indirect: Request to restore a snapshot via the /snapshots API.
+* Direct: Request to `/actions` API.
+* Indirect: Request to restore a snapshot via the `/snapshots` API.
 
 The dynamic portion of the life cycle is where the action is executed to completion. It begins when the actions is transitioned from the `new` to `configured` state. The action will then be ultimately transitioned to an end state of `aborted` or `completed`.
 
 <a name="firmware-operations"></a>
 
-### Firmware Operations
+## Firmware Operations
 
 Operations are individual tasks in a FAS action.
 FAS will create operations based on the configuration sent through the `actions create` command.
 FAS operations will have one of the following states:
-- initial - Operation just created.
-- configured - The operation is configured, but nothing has been started.
-- blocked - Only one operation can be performed on a node at a time. If more than one update is required for a component name (xname), operations will be blocked. This will have a message of "blocked by sibling".
-- inProgress - Update is in progress, but not completed.
-- verifying - Waiting for update to complete.
-- failed - An update was attempted, but FAS is unable to tell that the update succeeded in the allotted time.
-- noOperation - Firmware is at the correct version according to the images loaded into FAS.
-- noSolution - FAS does not have a suitable image for an update.
-- aborted - the operation was aborted before it could determine if it was successful. If aborted after the update command was sent to the node, the node may still have updated.
+
+* initial - Operation just created.
+* configured - The operation is configured, but nothing has been started.
+* blocked - Only one operation can be performed on a node at a time. If more than one update is required for a component name (xname), operations will be blocked. This will have a message of "blocked by sibling".
```

Comment:
```suggestion
* `blocked` - Only one operation can be performed on a node at a time. If more than one update is required for a component name (xname), then operations will be blocked. This will have a message of `blocked by sibling`.
```